booking-app/src/test/java/org/motechproject/bookingapp/service/ScreeningServiceTest.java

```diff
@@ -0,0 +1,161 @@
+package org.motechproject.bookingapp.service;
+
+import org.joda.time.LocalDate;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.motechproject.bookingapp.domain.Clinic;
+import org.motechproject.bookingapp.domain.Screening;
+import org.motechproject.bookingapp.domain.ScreeningDto;
+import org.motechproject.bookingapp.domain.Volunteer;
+import org.motechproject.bookingapp.exception.LimitationExceededException;
+import org.motechproject.bookingapp.helper.VisitLimitationHelper;
+import org.motechproject.bookingapp.repository.ClinicDataService;
+import org.motechproject.bookingapp.repository.ScreeningDataService;
+import org.motechproject.bookingapp.repository.VolunteerDataService;
+import org.motechproject.bookingapp.service.impl.ScreeningServiceImpl;
+import org.motechproject.commons.date.model.Time;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class ScreeningServiceTest {
+
+    @InjectMocks
+    private ScreeningService screeningService = new ScreeningServiceImpl();
+
+    @Mock
+    private ScreeningDataService screeningDataService;
+
+    @Mock
+    private VolunteerDataService volunteerDataService;
+
+    @Mock
+    private ClinicDataService clinicDataService;
+
+    @Mock
+    private VisitLimitationHelper visitLimitationHelper;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private Clinic clinic;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        clinic = new Clinic("siteId", "location", 1, 5, 2, 9, 8, 7, 10, 10, 10, 10, 10, 10, 10);
+        clinic.setId(1l);
+    }
+
+    @Test
+    public void shouldThrowNumberOfRoomsExceptionWhenNumberOfRoomsLimitIsReached() {
```

Comment:
we do not have NumberOfRoomsException, only LimitationExceededException, change the name of this test and all other tests with non existing exceptions in the name